uTinyRipperCore/Parser/FileCollection/Exporter/Exporters/Collections/AssetExportCollection.cs

```diff
@@ -35,23 +35,31 @@ public AssetExportCollection(IAssetExporter assetExporter, Object asset, IAssetI
 		}
 
 		public override bool Export(ProjectAssetContainer container, string dirPath)
-		{
-			string subFolder = Asset.ExportName;
+        {
+            string subFolder = Asset.ExportName;
 			string subPath = Path.Combine(dirPath, subFolder);
 			string fileName = GetUniqueFileName(container.File, Asset, subPath);
 			string filePath = Path.Combine(subPath, fileName);
 
-			if (!DirectoryUtils.Exists(subPath))
+            string resourceSubFolder = String.Empty;
+            string resourceFileName = String.Empty;
+            if (container.GetResourcePathFromAssets(Assets, filePath, ref resourceSubFolder, ref resourceFileName))
```

Comment:
I think it should be 'out string resourceSubFolder, out string resourceFileName'